include/deal.II/lac/utilities.h

```diff
@@ -147,6 +168,73 @@ namespace Utilities
       return diagonal[k-1] + f->l2_norm();
     }
 
+
+    template <typename OperatorType, typename VectorType>
+    void Chebyshev_filter(VectorType &x,
+                          const OperatorType &op,
+                          const unsigned int degree,
+                          const double a,
+                          const double b,
+                          const double a_L,
+                          VectorMemory<VectorType> &vector_memory)
+    {
+      Assert (degree > 0,
+              ExcNotImplemented());
```

Comment:
Maybe `ExcMessage ("Only positive degrees make sense.")`? I suspect that it's not that you didn't implement zero degrees yet, but that they don't make sense, right?